PFR/ForMathlib/FiniteRange.lean

```diff
@@ -109,8 +109,23 @@ lemma FiniteRange.full {Ω G : Type*} [MeasurableSpace Ω] [MeasurableSpace G] [
   ext ω; simp
   measurability
 
-lemma FiniteRange.null_of_compl {Ω G : Type*} [MeasurableSpace Ω] [MeasurableSpace G] [MeasurableSingletonClass G] {X : Ω → G} (hX: Measurable X) [FiniteRange X] (μ: Measure Ω) : (μ.map X) (FiniteRange.toFinset X: Set G)ᶜ  = 0 := by
+lemma FiniteRange.null_of_compl {Ω G : Type*} [MeasurableSpace Ω] [MeasurableSpace G]
+    [MeasurableSingletonClass G] {X : Ω → G} (hX: Measurable X) [FiniteRange X] (μ: Measure Ω) :
+    (μ.map X) (FiniteRange.toFinset X: Set G)ᶜ = 0 := by
   rw [Measure.map_apply hX]
   convert measure_empty
   ext ω; simp
   measurability
+
+lemma FiniteRange.null_of_compl₀ {Ω G : Type*} [MeasurableSpace Ω] [MeasurableSpace G]
+    [MeasurableSingletonClass G] (μ : Measure Ω)
+    {X : Ω → G} (hX : AEMeasurable X μ) [FiniteRange X] :
```

Comment:
You don't need `hX` here: otherwise, `mu.map X` is the zero measure, so the conclusion is obviously true.